src/main/java/se/liu/ida/hefquin/engine/federation/SPARQLEndpoint.java

```diff
@@ -1,9 +1,13 @@
 package se.liu.ida.hefquin.engine.federation;
 
+import se.liu.ida.hefquin.engine.data.VocabularyMapping;
 import se.liu.ida.hefquin.engine.federation.access.SPARQLEndpointInterface;
 
 public interface SPARQLEndpoint extends FederationMember
 {
 	@Override
 	SPARQLEndpointInterface getInterface();
+	
+	@Override
+	VocabularyMapping getVocabularyMapping();
```

Comment:
That's not necessary and should be removed again. Sub-interfaces inherit all the functions of their parent interface.

(Now, you may wonder why we repeat the `getInterface()` function here in the sub-interface because that one is also already declared in the parent interface. The reason is that, for this sub-interface, we restrict the return type of that function.)